### ISSUE TITLE
Expression "@reboot" does not work

### ISSUE THREAD

**Issue by jcassee** [ahawker/crython#34]
The expression `@reboot` does not work. Running `crython.start()` outputs this error:

```
Traceback (most recent call last):
[...
  File "/home/joost/Ontwikkeling/GoAbout/Docker/tarsnap/crython/local/lib/python2.7/site-packages/crython/job.py", line 39, in decorator
    wrapper.cron_expression = expression.CronExpression.new(expr, **fields)
  File "/home/joost/Ontwikkeling/GoAbout/Docker/tarsnap/crython/local/lib/python2.7/site-packages/crython/expression.py", line 116, in new
    return cls.from_str(expression) if expression else cls.from_kwargs(**kwargs)
  File "/home/joost/Ontwikkeling/GoAbout/Docker/tarsnap/crython/local/lib/python2.7/site-packages/crython/expression.py", line 127, in from_str
    fields = _expression_str_to_dict(expression, reboot_sentinel=reboot_sentinel)
  File "/home/joost/Ontwikkeling/GoAbout/Docker/tarsnap/crython/local/lib/python2.7/site-packages/crython/expression.py", line 69, in _expression_str_to_dict
    raise ValueError('Expression contains {0} fields; expects {1}'.format(len(values), expression_field_count))
ValueError: Expression contains 1 fields; expects 7
```

The cause is probably the line `if expression is REBOOT_KEYWORD` in `expression.py` which should be `if expression == REBOOT_KEYWORD`.

**Comment by ahawker** [comment]
@jcassee 

Thanks for reporting this and sorry for the issue. I'm currently out on vacation but will take a deeper look once I'm back at a computer regularly.

**Comment by jcassee** [comment]
No problem, I have a work-around:

```
if schedule == '@reboot':
    from crython.expression import REBOOT_KEYWORD
    schedule = REBOOT_KEYWORD
```

**Comment by ahawker** [comment]
Fix should be available in version `0.0.7`.

**Comment by jcassee** [comment]
Great, thanks!

**Comment by zalum** [comment]
Hello @ahawker ,

also in 0.0.7 I get by running
`print(crython.__version__)
@crython.job(expr='@reboot')
def test():
    print("hmm")`

I get
`0.0.7
Traceback (most recent call last):
  File "/data/git/florinz/system-model/zooplus-diagram-view/zooplus_system_view_web.py", line 3, in <module>
    from datamodel.oracle_datamodel_collector import start_collector
  File "/data/git/florinz/system-model/zooplus-diagram-view/datamodel/oracle_datamodel_collector.py", line 22, in <module>
    @crython.job(expr='@reboot')
  File "/data/tools/anaconda/lib/python3.6/site-packages/crython/job.py", line 39, in decorator
    wrapper.cron_expression = expression.CronExpression.new(expr, **fields)
  File "/data/tools/anaconda/lib/python3.6/site-packages/crython/expression.py", line 116, in new
    return cls.from_str(expression) if expression else cls.from_kwargs(**kwargs)
  File "/data/tools/anaconda/lib/python3.6/site-packages/crython/expression.py", line 127, in from_str
    fields = _expression_str_to_dict(expression, reboot_sentinel=reboot_sentinel)
  File "/data/tools/anaconda/lib/python3.6/site-packages/crython/expression.py", line 69, in _expression_str_to_dict
    raise ValueError('Expression contains {0} fields; expects {1}'.format(len(values), expression_field_count))
ValueError: Expression contains 1 fields; expects 7
`

**Comment by ahawker** [comment]
@zalum 

Apologies for the confusion, this fix was actually released in `0.0.8`, not `0.0.7`. I've updated the comment above in this issue to address it.

The commit that fixed this is https://github.com/ahawker/crython/commit/3ddd56e5c4fdab453aad84f3a0c7c0a7df8c70f5

Reproduction:

```
⇒  cat issue34.py
import crython

print(crython.__version__)
@crython.job(expr='@reboot')
def test():
    print("hmm")
```

Version: 0.0.7

```
⇒  git checkout v0.0.7
Note: checking out 'v0.0.7'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 422ab25... Bump version: 0.0.6 → 0.0.7
hawker@mbp:~/src/github.com/ahawker/crython|422ab25⚡
⇒  python issue34.py
0.0.7
Traceback (most recent call last):
  File "issue34.py", line 4, in <module>
    @crython.job(expr='@reboot')
  File "/Users/hawker/src/github.com/ahawker/crython/crython/job.py", line 39, in decorator
    wrapper.cron_expression = expression.CronExpression.new(expr, **fields)
  File "/Users/hawker/src/github.com/ahawker/crython/crython/expression.py", line 116, in new
    return cls.from_str(expression) if expression else cls.from_kwargs(**kwargs)
  File "/Users/hawker/src/github.com/ahawker/crython/crython/expression.py", line 127, in from_str
    fields = _expression_str_to_dict(expression, reboot_sentinel=reboot_sentinel)
  File "/Users/hawker/src/github.com/ahawker/crython/crython/expression.py", line 69, in _expression_str_to_dict
    raise ValueError('Expression contains {0} fields; expects {1}'.format(len(values), expression_field_count))
ValueError: Expression contains 1 fields; expects 7
```

Version 0.0.8:

```
⇒  git checkout v0.0.8
Note: checking out 'v0.0.8'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b6c17e1... Bump version: 0.0.7 → 0.0.8
hawker@mbp:~/src/github.com/ahawker/crython|b6c17e1⚡
⇒  python issue34.py
0.0.8
```
